index.html

```diff
@@ -0,0 +1,199 @@
+                  </div>
+                </div>
+              </div>
+            </article>
+            <article class="card program_item border-0">
+              <div class="row g-0 d-flex">
+                <div class="col-2">
+                  <img src="images/forum_icon_03 1.svg" class=" rounded-start program_icon" alt="forum">
+                </div>
+                <div class="col-3">
+                  <div class="card-title" style="max-width: auto;">
+                    <h5 class="program_items_title">Forum</h5>
+                  </div>
+                </div>
+                <div class="col-7">
+                  <div class="card-body">
+                    <p class="card-text program_items_content">Have the time to share your thoughts and opinions with experts for each topic.</p>
+                  </div>
+                </div>
+              </div>
+            </article>
+            <article class="card program_item border-0">
+              <div class="row g-0 d-flex">
+                <div class="col-2">
+                  <img src="images/workshop_icon_04 1.svg" class=" rounded-start program_icon" alt="Workshop">
+                </div>
+                <div class="col-3">
+                  <div class="card-title" style="max-width: auto;">
+                    <h5 class="program_items_title">Workshop</h5>
+                  </div>
+                </div>
+                <div class="col-7">
+                  <div class="card-body">
+                    <p class="card-text program_items_content">Try creating your own work using open source license rather than just watching at it.</p>
+                  </div>
+                </div>
+              </div>
+            </article>
+            <article class="card program_item border-0">
+              <div class="row g-0 d-flex">
+                <div class="col-2">
+                  <img src="images/ignite_icon_05 1.svg" class=" rounded-start program_icon" alt="Ignite">
+                </div>
+                <div class="col-3">
+                  <div class="card-title" style="max-width: auto;">
+                    <h5 class="program_items_title">Opti Ignite</h5>
+                  </div>
+                </div>
+                <div class="col-7">
+                  <div class="card-body">
+                    <p class="card-text program_items_content">get opportunities to network with CC affiliates around the world, also after the summit.</p>
+                  </div>
+                </div>
+              </div>
+            </article> 
+        </section> 
+        <button type="button" class="btn btn btn-danger btn-lg rounded-0" id="btn_join">Join Opti Lion exhibition</button>
+        <a href="#" class="see_program">SEE THE WHOLE PROGRAM</a>
+    </section>
+    <section id="featured_speaker" class="d-flex flex-column align-items-center">
```

Comment:
- [ ] Kindly fix the breakpoint issue from desktop size to tablet size the image is not showing well. 
<img width="951" alt="Screenshot 2023-01-06 at 15 11 13" src="https://user-images.githubusercontent.com/81444338/211115519-d477c96d-8fd0-4567-97bc-b995930a8599.png">